docs/release-notes/2.40.1.md

```diff
@@ -0,0 +1,20 @@
+# Percona Monitoring and Management 2.40.1
+
+| **Release date:** | Oct 18, 2023                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+## Fixed issues
+
+- [PMM-12592](https://jira.percona.com/browse/PMM-12592) - Fixed an issue where the **Library Panels** for the PMM dashboard were not working.
+- [PMM-12576](https://jira.percona.com/browse/PMM-12576) - After upgrading to PMM version 2.40.0, changing the Admin user's password from the terminal was not functioning. The issue has been resolved now. This impacts all versions prior to PMM 2.40.0.
```

Comment:
"This impacts all versions prior to PMM 2.40.0." 
Is this?? 
I'm reading this as "in all versions before 2.40  user is not able to change the password. " and this is not the case, afaik

Reply:
let's remove part about impact

Reply:
ok